### DISCUSSION TITLE
Exception invalid uuid

### DISCUSSION THREAD

**Discussion by trevorpan** [spatie/laravel-medialibrary#2240] [Q&A] [answered]
```
//vue component

<template>
        <div id="jobFileUploader">
            <media-library-attachment
                name="documents"
                :max-items="10"
                :validation-rules="{ accept: ['image/png', 'image/jpeg', 'application/pdf'], maxSizeInKB: 15360 }"
                :validation-errors="$errors"

                multiple
                @is-ready-to-submit-change="isReadyToSubmit = $event"
            />
            <div class="form-group">
                <button :disabled="isReadyToSubmit" type="submit" class="btn btn-warning mt-3">
                    Add new media files
                </button>
            </div>
        </div>
</template>

<script>
import { MediaLibraryAttachment } from "media-library-pro-vue2-attachment";

export default {
    components: { MediaLibraryAttachment },

    data() {
        return {
            isReadyToSubmit: true,
        };
    },
};
</script>

```
I'm having an issue where the vue component button is active while uploading but disabled when ready to submit. Referencing this (v3, I'm using v2)
https://spatie.be/docs/laravel-medialibrary/v9/handling-uploads-with-media-library-pro/handling-uploads-with-vue#checking-the-upload-state

What would cause this component to work opposite of intended behavior?

**Comment by trevorpan** [comment, marked as answer]
After much head banging....

It was a simple `!` added to  `:disabled="!isReadyToSubmit"`

```
<div class="form-group">
    <button v-bind:disabled="!isReadyToSubmit" type="submit" class="btn btn-warning mt-3"> 
        Add new media files
    </button>
</div>
```
@freekmurze Is this something you'd like to add to the docs? I can do a PR.

**Comment by freekmurze** [comment]
@trevorpan Feel free to send a PR to the docs.

@AdrianMrn is the solution above ok for you?

**Reply by trevorpan** [reply]
Will wait for @AdrianMrn  the only thing I'm not 100% is it may not need `v-bind:` Otherwise, it's super pro!

Basically, I'm reading it like: button is disabled unless it's ready to submit.

**Reply by AdrianMrn** [reply]
Whoops, my bad! 😬 looks like it was correct in the React docs, but not the Vue docs. The double negation must have tripped me up 🙈  Thanks for the [PR](https://github.com/spatie/laravel-medialibrary/commit/1284ec3889c8ff730b9c2238db2ef07499c0ada7)!

**Reply by trevorpan** [reply]
You, bet. Appreciate the work put in this package. Very slick!
